### PR TITLE
Post feed and create post page integrated

### PR DESCRIPTION
**Pull request by shazebK**
@RitwikSingh28 
I have done some style related changes in postfeed.dart and created the functionality to create a post by the user which later can be visible on the postfeed page.

Please review this PR and merge 😊.

postfeed

### CODE REVIEW

File: lib/main.dart [hunk 1]
```diff
@@ -28,9 +27,7 @@ class ScreenDecider extends StatelessWidget {
         primarySwatch: Colors.blue,
       ),
       debugShowCheckedModeBanner: false,
-      initialRoute: auth.getCurrentUser() == null
```
Comment (RitwikSingh28) [inline lib/main.dart:31]: @shazebK can you reset the initialRoute to its initial config please? Thanks! :+1:

**Review by RitwikSingh28 — CHANGES_REQUESTED** [review]
Splendidly done! :sparkler: 

Please just make one change, and after that we would be good to go

**Comment by shazebK** [comment]
Ok I will try

**Comment by shazebK** [comment]
@RitwikSingh28 
I have made the necessary changes as requested

**Review by RitwikSingh28 — APPROVED** [review]
No issues now... Merging!

**Pull request merged**